qkit/drivers/Agilent_VNA_E5071C.py

```diff
@@ -724,15 +741,15 @@ def start_measurement(self):
         This function is called at the beginning of each single measurement in the spectroscopy script.
         Here, it resets the averaging.  
         '''
-        self._visainstrument.write(':TRIG:AVER ON')
+        self.write(':TRIG:AVER ON')
```

Comment:
If I remember correctly, @MartinSpiecker had some problems with that command. Their (new) VNAs were ignoring the trigger afterwards.